src/tests/rpp/test_flat_map.cpp

```diff
@@ -105,22 +106,43 @@ TEMPLATE_TEST_CASE("flat_map", "", rpp::memory_model::use_stack, rpp::memory_mod
                 CHECK(mock.get_on_error_count() == 1);
             }
         }
+        SECTION("subscribe using flat_map with templated lambda")
```

Comment:
i think, you can just place it in https://github.com/victimsnino/ReactivePlusPlus/pull/374/commits/a748a6f2997d75f16f5e5998d75f83696543da7d#diff-4ad61bca345fc2ef117d5617e164dca71340d801c636340f2c051a5bd4e6e9d2R37

Reply:
done